src/ui/pages/Menu/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -53,7 +52,9 @@ const ConnectWallet = forwardRef<ConnectWalletOptionRef, object>(
 
     const toastMsg = useAppSelector(getToastMsg);
     const pendingConnection = useAppSelector(getPendingConnection);
-    const identifierCache = useAppSelector(getIdentifiersCache);
+    const identifierCache = useAppSelector(getIdentifiersCache).filter(
+      (identifier) => !identifier.multisigManageAid
```

Comment:
Actually, we also shouldn't show the local member identifiers of a multi-sig. So we should filter for `groupMetadata too. And maybe re-name. Please update the test too then

```
const defaultIdentifierCache = useAppSelector(getIdentifiersCache).filter(
  (identifier) => !identifier.multisigManageAid && !identifier.groupMetadata
);
```